.mage/magefile.go

```diff
@@ -25,15 +27,36 @@ import (
 	"go.einride.tech/mage-tools/targets/mggoreview"
 )
 
+func init() {
+	mgmake.AddExtraMakefiles(
+		[]mgmake.Makefile{
+			{
+				Path:          mgtool.GetGitRootPath("proto/Makefile"),
+				Namespace:     Proto{},
+				DefaultTarget: "Hej",
+			},
+		},
+	)
+}
+
+var Default = All
```

Comment:
Maybe we should just call it `Default` to avoid having this extra variable?

Reply:
Or maybe setting the `.DEFAULT_TARGET` is done by a method call to the `mgmake` package instead of a magic variable?

Reply:
We could change the entire logic, and instead of automatically creating a `Makefile` in the root of the git repo, that file too needs to be defined, similar to how we define the proto makefile? That would potenially make stuff a bit more clear too the end user?